File: IRepoAngajat.java
package me.repository;

import me.model.Angajat;

import java.util.List;

public interface IRepoAngajat extends Repository<Integer,Angajat>{
    List<Angajat> findAngajatiNume(String nume);
    Angajat getUsernameParola(String username, String parola);
}

File: Concurs Inot C#/IRepoAngajat.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Problema_3_CSharp.me.model;

namespace Problema_3_CSharp.me.repository
{
    public interface IRepoAngajat : IRepository<int,Angajat>
    {
        Angajat GetAngajatUsernameParola(string username, string parola);
        IEnumerable<Angajat> FindAngajatiDupaNume(string nume);
    }
}
